src/main/java/com/volmit/iris/engine/object/IrisObjectPlacement.java

```diff
@@ -126,6 +126,10 @@
     @Desc("This object / these objects override the following trees when they grow...")
     @ArrayType(min = 1, type = IrisTree.class)
     private KList<IrisTree> trees = new KList<>();
+    @Desc("Run raw commands at the location of this object")
+    @ArrayType(min = 1, type = IrisCommand.class)
+    private KList<IrisCommand> rawCommands = new KList<>();
```

Comment:
Nope, Iris objects never place in-world they place in the mantle this would either throw a bunch of errors and do nothing, or just do nothing.